aspnetcore/fundamentals/openapi/buildtime-openapi.md

```diff
@@ -8,18 +8,19 @@ ms.custom: mvc
 ms.date: 8/13/2024
```

Comment:
Question for @captainsafia: This doc currently says aspnetcore-6.0 and later. Should we make this just 9.0 and later, since 8.0 and earlier involve Swashbuckle or NSwag generating the doc?

Reply:
> Question for @captainsafia: This doc currently says aspnetcore-6.0 and later. Should we make this just 9.0 and later, since 8.0 and earlier involve Swashbuckle or NSwag generating the doc?

Yeah, let's do it.

I think it'll make managing this page a little bit easier if we make updates to the doc process a bit later.